packaging/templates/package_scripts/postinst

```diff
@@ -2,6 +2,11 @@
 # This is a postinstallation script so the service can be configured and started when requested
 #
 sudo adduser --disabled-password --disabled-login --shell /usr/sbin/nologin --quiet --system --no-create-home --home /nonexistent bor
```

Comment:
If the user already exists, will this fail? Maybe it is better to check if user `bor` exists before adding it?

Reply:
Looks like this command won't fail even when the user already exists.